### PR TITLE
Use the correct login help message based on Juju version.

### PR DESCRIPTION
**Pull request by frankban**
It should be ok to QA this just on sandbox, changing the config options to simulate jujuCoreVersion == 1.x.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4527/
Test PASSed.

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -693,10 +693,17 @@ YUI.add('juju-gui', function(Y) {
       @param {Boolean} failure Whether the login failed.
     */
     _renderLogin: function(failure) {
+      var msg = 'Find your password with ' +
+                '`juju show-controller --include-passwords`';
```
Comment (bac) [inline jujugui/static/gui/src/app/app.js]: With juju 2.0, the username is now 'admin@local', which is shown by the command.  So maybe say 'Find your username and password'

**Comment by huwshimi** [comment]
This looks great @frankban. QA OK in Juju 1.25.3-wily-amd64, but someone with a compiled Juju 2.0 will need to check that. :+1:

File: jujugui/static/gui/src/app/app.js (hunk 1 above)
Comment (frankban) [inline jujugui/static/gui/src/app/app.js]: Good point, done.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4534/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4535/
Test FAILed.

**Comment by frankban** [comment]
Thanks for the reviews! I QAed it in Juju 2.0 and the message is properly displayed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4536/
Test PASSed.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**